weather/script.py

```diff
@@ -1,10 +1,17 @@
+
```

Comment:
please, delete empty line, at the beggining

Reply:
Thanks, will remove in next commit